src/libUtils/SysCommand.h

```diff
@@ -19,17 +19,89 @@
 #define __SYSCOMMAND_H__
 
 #include <signal.h>
+#include <sys/wait.h>
 #include <array>
 #include <cstdlib>
 #include <fstream>
 #include <iostream>
 #include <memory>
 #include <string>
 
+#include <boost/algorithm/string/classification.hpp>
+#include <boost/algorithm/string/split.hpp>
+
 #include "Logger.h"
 
+#define READ 0
+#define WRITE 1
+
+static std::string cmd_output_place_holder;
+static int pid_place_holder;
+
 class SysCommand {
  public:
```

Comment:
The modification is fine. Since now only ExecuteCmd() be used, how about make other functions private?

Reply:
static can't call member function, unless we make the syscommand a singleton class

Reply:
ok